fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -489,8 +604,16 @@ def initialize(user = nil, password = nil, use_shell_script = false, provider_sh
       end
 
       @jwt = jwt
+      @api_key = api_key
+
+      if should_use_altool?(upload, use_shell_script)
+        UI.verbose("Use altool as transporter.")
```

Comment:
Maybe `Using` makes more sense here:

```suggestion
        UI.verbose("Using altool as transporter.")
```